src/ClientBuilder.php

```diff
@@ -299,12 +299,12 @@ public function __call($name, $arguments)
      */
     private function createHttpClientInstance()
     {
-        if (!$this->uriFactory instanceof UriFactory) {
-            throw new \RuntimeException('UriFactory is not ready');
+        if (null === $this->uriFactory) {
+            throw new \RuntimeException('The PSR-7 Uri factory must be set');
         }
 
         if (null === $this->httpClient) {
-            throw new \RuntimeException('HttpClient is not ready');
+            throw new \RuntimeException('The PSR-18 Http client must be set');
```

Comment:
HTTP is an acronym and should be uppercase. Also please add a dot at the end of the sentence